props/prop_base.h

```diff
@@ -542,17 +542,32 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   bool blade_detected_ = false;
 #endif
 
+  // Prevent BLADE ID: from spamming console with every BLADE_SCAN_MILLIS
+  bool blade_id_should_report_ = true;
+
   // Measure and return the blade identifier resistor.
   float id() {
     EnableBooster();
     BLADE_ID_CLASS_INTERNAL blade_id;
     float ret = blade_id.id();
+#ifdef BLADE_ID_SCAN_MILLIS
```

Comment:
Nesting #Ifdefs and duplicating code is not very clean.

Reply:
gone.